pkg/kubelet/kubelet.go

```diff
@@ -1722,12 +1752,41 @@ func (kl *Kubelet) makePodDataDirs(pod *api.Pod) error {
 	return nil
 }
 
+// syncPod is the transaction script for the sync of a single pod.
+//
+// Arguments:
+//
+// pod - the pod to sync
+// mirrorPod - the mirror pod for the pod to sync, if it is a static pod
+// podStatus - the current status (TODO: always from the status manager?)
+// updateType - the type of update (CREATE,ADD,DELETE,RECONCILE)
+//
+// The workflow is:
+//  1.  If the pod is being created, record pod worker start latency
```

Comment:
I'm not sure if documenting how we record the latencies in details here is necessary. This might change soon and is not really the core of syncPod's functionality.